environments/linux/docker/.env

```diff
@@ -0,0 +1,2 @@
+### Set default arch
+DEFAULT_ARCH=linux/amd64
```

Comment:
Could we determine this via `dpkg --print-architecture`

Reply:
Unless we make a script to launch the docker compose bits, I'd rather see the runner actually request the different arch with `DEFAULT_ARCH=linux/arm64 docker compose blahhh`. 

This allows QA to make changes where they need to.